recipe/meta.yaml

```diff
@@ -1,20 +1,18 @@
 {% set name = "trio" %}
-{% set version = "0.12.1" %}
-{% set hash_type = "sha256" %}
-{% set hash_value = "149a280a4c66ef08d058626e3f48bf979e6e079777d1fc5cae71676586ced672" %}
+{% set version = "0.16.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  {{ hash_type }}: {{ hash_value }}
+  sha256: df067dd0560c321af39d412cd81fc3a7d13f55af9150527daab980683e9fcf3c
```

Comment:
I'd like to keep the hash parametrized on top of the file, if possible. That way bumping a version number appears on one chunk.

Reply:
I'll look into where this change takes place today.

Reply:
Thanks!

Reply:
So I found where it makes this change [here](https://github.com/regro/cf-scripts/blob/3a757fcc27806e882f497e3b79ce7f77b9054805/conda_forge_tick/migrators/extra_jinj2a_keys_cleanup.py#L12). It says that it makes this change to make it easier for the version migrator to do its job. I can try excluding the `ExtraJinja2KeysCleanup` migrator and see what happens if you want.

Reply:
OK, sounds good, it's time to experiment!